### ISSUE TITLE
null, not nil

### ISSUE THREAD

**Issue by kwerle** [adrienverge/yamllint#382]
We're.a ruby shop - and in ruby we say nil instead of null.  We're special!

Unfortunately, if you put nil in a yaml file, it thinks you mean "nil".  Which we NEVER mean.

So I'd love to see a nil check as follows:

BAD
some_key: nil

GOOD
some_key: "nil"

That way when some ruby developer drops a nil into a yaml they get caught. And if, for some crazy reason, they mean it - just put it in quotes.

**Comment by adrienverge** [comment]
Hello,

yamllint enforces rules from the [YAML specification](https://yaml.org/spec/1.2/spec.html), and `nil` isn't a special keyword in YAML.

But I think you can achieve what you want using the `quoted-strings` rule. I let you read the documentation: https://yamllint.readthedocs.io/en/latest/rules.html#module-yamllint.rules.quoted_strings

**Comment by kwerle** [comment]
That looks like it'd do the trick - thanks!

**Comment by adrienverge** [comment]
> That looks like it'd do the trick - thanks!

Great!

Maybe you can make it harder with `^nil$` (to avoid requiring quotes for `anilz`) and simpler by adding quotes (to remove the comment):
```yaml
  quoted-strings:
    required: false
    extra-required: ['^nil$']
```
